### ISSUE TITLE
Publishing MoT API issues

### ISSUE THREAD

**Issue by dkurepa** [dotnet/arcade#10069]
I just wanted to chronicle my journey when publishing the MoT API to dotnet-eng

**Comment by dkurepa** [comment]
I used publishing of the Microsoft.Internal.Helix.Admin.CommandLine to see how to publish this API. Originally, I copied the whole csproj and just changed the naming. I tried using dotnet pack to create a nuget from the project, but it didn't work, because the API was a netstandard2.1 app. Because of this I changed it to a netcoreapp3.1. This fixed the problem, and I was able to use dotnet pack. I uploaded this nuget to the dotnet-eng feed, and tested it in a helix-service project. It didn't work, the versions/project types were not compatible. I thought this might be because 3.1 wasn't compatible with .net 6. I also tried using the nuget with a random 3.1 project and it didn't work again. After this I knew something was wrong with the way I was producing the nuget. After some investigation I noticed I copied <PackAsTool>true</PackAsTool> from the CommandLine tool. Deleting this solved the issue, the nuget now works in all projects
